remotespark/livyclientlib/sparkcontroller.py

```diff
@@ -72,6 +72,7 @@ def add_session(self, name, connection_string, skip_if_exists, properties):
         session.start()
         livy_client = self.client_factory.build_client(session)
         self.client_manager.add_client(name, livy_client)
+        livy_client.start()
```

Comment:
I don't have strong feelings on this. It just might be useful to have the factory call the start method because then you are not calling it everywhere.